modRlp/test/org/aion/rlp/RLPSpecExtraTest.java

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.rlp;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.aion.rlp.RLPSpecTest.assertDecodeBigInteger;
+import static org.aion.rlp.RLPSpecTest.assertDecodeInt;
+import static org.aion.rlp.RLPSpecTest.assertDecodeLong;
+import static org.aion.rlp.RLPSpecTest.assertEncodeBigInteger;
+import static org.aion.rlp.RLPSpecTest.assertEncodeInt;
+import static org.aion.rlp.RLPSpecTest.assertEncodeLong;
+import static org.aion.rlp.RLPSpecTest.assertEncodeShort;
+
+import java.math.BigInteger;
+import org.aion.base.util.Hex;
+import org.junit.Test;
+
+/**
+ * Tests for the RLP implementation that are complementary to RLP specification tests from Ethereum
+ * implemented in {@link RLPSpecTest}.
+ *
+ * @author Alexandra Roatis
+ */
+public class RLPSpecExtraTest {
+
+    // ENCODING
+
+    @Test
+    public void testEncodeShort3() {
```

Comment:
Can we also test the max value for `short` and the min values for `short`,`int`,`long` (looks like our min is zero?) just to have those edges covered

Reply:
That's covered in the `RLPSpecTest.java` which is based on [rlptest.json](https://github.com/ethereum/tests/blob/develop/RLPTests/rlptest.json). These tests complement those test cases with values that I considered may be relevant.